tests/integration_tests/utils_tests.py

```diff
@@ -85,19 +85,6 @@
 from .fixtures.certificates import ssl_certificate
 
 
-def mock_to_adhoc(filt, expressionType="SIMPLE", clause="where"):
-    result = {"clause": clause.upper(), "expressionType": expressionType}
-
-    if expressionType == "SIMPLE":
-        result.update(
-            {"comparator": filt["val"], "operator": filt["op"], "subject": filt["col"]}
-        )
-    elif expressionType == "SQL":
-        result.update({"sqlExpression": filt[clause]})
-
-    return result
-
-
```

Comment:
Having a mock like this felt dangerous, as it's mostly the same as the original function + the logic associated with conversion between regular and adhoc metrics is already brittle. Therefore this mock is removed and the assertions updated to reflect what is actually expected to be returned when running the full chain of `convert_legacy_filters_into_adhoc`,  `merge_extra_filters` and `split_adhoc_filters_into_base_filters`.